components/mediation-admin/org.wso2.carbon.endpoint/src/main/java/org/wso2/carbon/endpoint/service/EndpointAdmin.java

```diff
@@ -69,6 +70,8 @@
 
     private static final Log log = LogFactory.getLog(EndpointAdmin.class);
     public static final String WSO2_ENDPOINT_MEDIA_TYPE = "application/vnd.wso2.esb.endpoint";
+    private boolean saveRuntimeArtifacts =
+            SynapsePropertiesLoader.getBooleanProperty("synapse.runtime_artifacts.save.local.file", true);
```

Comment:
Shall we define a const in SynapseConstants and use it here. Also shall we have some other better prop name since boolean ( e:- synapse.artifacts.file.storage.enabled ) wdyt ?